infra/blueprint-test/pkg/kpt/kpt.go

```diff
@@ -57,6 +61,17 @@ func NewCmdConfig(t testing.TB, opts ...cmdOption) *CmdCfg {
 		}
 		kOpts.kptBinary = "kpt"
 	}
+	// Validate required KPT version
+	kptVersion, err := utils.KptVersion(kOpts.kptBinary)
```

Comment:
It would be good to keep the kpt specific logic in this package. Couldn't we do `kOpts.RunCmd("version")` to get the version here and maybe the utils could have a general `minSemver(t testing.TB, gotSemver, minSemver string)` func which we call here to assert gotSemver >= minSemver.

Reply:
@bharathkkb - Good idea, I think this is a lot cleaner/reusable.  Let me know if you have anything else before I merge.